sdks/python/apache_beam/transforms/util.py

```diff
@@ -223,7 +223,7 @@ def __init__(self,
     if target_batch_duration_secs and target_batch_duration_secs <= 0:
       raise ValueError("target_batch_duration_secs (%s) must be positive" % (
           target_batch_duration_secs))
-    if not (target_batch_overhead or target_batch_duration_secs):
```

Comment:
This line should be fine to keep.

Reply:
Yes we can keep this line. The roll-back PR was auto-generated and merged. This is a CP of the rollback to release branch. I don't it would make a difference  on the release whether we roll this line forward or not, but we can roll it forward on master as a part of our Python 3 conversion efforts.